x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/views/results_per_page_view.tsx

```diff
@@ -16,12 +16,12 @@ const wrapResultsPerPageOptionForEuiSelect: (option: number) => EuiSelectOption
 });
 
 interface Props {
-  options: number[];
-  value: number;
+  options?: number[];
+  value?: number;
   onChange(value: number): void;
 }
 
-export const ResultsPerPageView: React.FC<Props> = ({ onChange, options, value }) => {
+export const ResultsPerPageView: React.FC<Props> = ({ onChange, options = [], value = 20 }) => {
```

Comment:
Search UI says that these props are optional but that might be incorrect, added to issue to investigate: https://github.com/elastic/enterprise-search-team/issues/2226